learn/quickstart/arrays_strings.md

```diff
@@ -149,3 +149,34 @@ program allocatable_string
 end program allocatable_string
 ```
 
+
+## Array of strings
+
+The length of all strings in the array must be the same. The length assigned is typically the length of the longest element.
```

Comment:
I thought the explanation needed some clarification, I hope you like it.

```suggestion
An array of strings can be expressed in Fortran as an array of `character` variables.
All elements in a `character` array have equal length.
However, strings of varying lengths can be provided as input to the array constructor, as shown in the example below.
They will be truncated or right-padded with spaces if they are longer or shorter, respectively, than the declared length of the `character` array.
If the `character` array is declared as deferred-length (`character(len=:), allocatable`) instead, the `character` array will be allocated with the length of the longest element.
```

Reply:
Sorry, noted a spelling error after resolving - line 158, They will be truncated ... th(a)n the declared length ...

Reply:
Good catch, thanks. I noticed that you resolved the suggestions without committing them. Did you mean to do that?